common/internal_model/src/test/scala/weco/catalogue/internal_model/index/SearchIndexConfigJsonTest.scala

```diff
@@ -68,6 +72,14 @@ class SearchIndexConfigJsonTest
         indexName = index.name
       )
 
+      implicit val lcs: Patience[Json] = new Patience[Json]
+
+      println("Mapping diff:")
+      println(diff(fileMapping, indexMapping))
+
+      println("Settings diff:")
+      println(diff(fileMapping, indexMapping))
```

Comment:
This is so the test doesn't tell us "(1500 lines of JSON) is different to (1500 lines of JSON)". It's not super helpful, but it at least gives a clue as to where to look.